code/src/App.js

```diff
@@ -1,12 +1,27 @@
 import React from 'react'
 import data from './data.json'
 
-console.log(data)
+import Album from './components/Album'
+import Header from './components/Header'
+
+console.log('data',data)
+
 
 export const App = () => {
   return (
-    <div>
-      Find me in src/app.js!
+   <div> 
+    <Header/> 
+    <section className='album-wrapper' >
+      {
```

Comment:
Very nice and clean App.js, according to the guidelines on clean entry points! Another thing you could do if you have the time is to move the map() out of the app.js to its own component